raft.go

```diff
@@ -748,7 +750,10 @@ func (r *Raft) runLeader() {
 
 	// Setup leader state
 	r.leaderState.commitCh = make(chan struct{}, 1)
-	r.leaderState.inflight = newInflight(r.leaderState.commitCh)
+	r.leaderState.commitment = newCommitment(r.leaderState.commitCh,
+		append([]string{r.localAddr}, r.peers...),
+		r.getLastIndex()+1)
```

Comment:
I'm probably not understanding something subtle - why does this increment by 1 here?